src/util/Permissions.js

```diff
@@ -132,9 +135,13 @@ Permissions.FLAGS = {
   USE_APPLICATION_COMMANDS: 1n << 31n,
   REQUEST_TO_SPEAK: 1n << 32n,
   MANAGE_THREADS: 1n << 34n,
+  // TODO: Remove deprecated USE_*_THREADS flags in v14
   USE_PUBLIC_THREADS: 1n << 35n,
+  CREATE_PUBLIC_THREADS: 1n << 35n,
   USE_PRIVATE_THREADS: 1n << 36n,
+  CREATE_PRIVATE_THREADS: 1n << 36n,
```

Comment:
Just one question:

With this patch, what does `new Permissions(3n << 35n).toArray()` return? `['USE_PUBLIC_THREADS', 'USE_PRIVATE_THREADS']` or `['CREATE_PUBLIC_THREADS', 'CREATE_PRIVATE_THREADS']`?

Reply:
@kyranet neither. It actually returns:
```js
[
  'USE_PUBLIC_THREADS',
  'CREATE_PUBLIC_THREADS',
  'USE_PRIVATE_THREADS',
  'CREATE_PRIVATE_THREADS'
]
```

Reply:
Isn't this actually breaking? @discordjs/the-big-3

Reply:
I don't understand how flag `1n << 35n` can return both public and private... Other than that..I think it should be fine?

Reply:
Not really, at least in my bot (which relies on `Permissions#toArray` for a few things), this would cause duplicates to happen, which can be troublesome, that's why I think this change is `semver: major`.

The rest of the changes in this PR are fine and fit in either `semver: patch` or `semver: minor`.

Reply:
This scenario already happened once.
We used to have aliases (due to name changes) for `USE_EXTERNAL_EMOJIS` and `MANAGE_ROLES` until the old names were removed with v12.
(Introduced with https://github.com/discordjs/discord.js/commit/16fe48d405f4dd9c40a68737b05aedc3f4f0d201)
I do not remember any trouble around these. Therefore I suppose it's fine to add these, not that they are strictly needed.